core/src/main/java/brooklyn/entity/rebind/RebindContextLookupContext.java

```diff
@@ -60,6 +66,9 @@ public RebindContextLookupContext(ManagementContext managementContext, RebindCon
     @Override public Entity lookupEntity(String id) {
         Entity result = rebindContext.getEntity(id);
         if (result == null) {
+            result = managementContext.lookup(id, Entity.class);
```

Comment:
But neither is the constructor to set `lookInManagementContext` to true. Can we just delete that?